portal-api/src/v1/users/sign-in-link.service.create-and-email-sign-in-link.test.js

```diff
@@ -109,7 +109,7 @@ describe('SignInLinkService', () => {
 
           testCreatingAndEmailingTheSignInLinkRejects({
             expectedCause: createSignInLinkTokenError,
```

Comment:
It's also surprising that this test is still passing after changing how we handle error causes